apps/pie-microsite/CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v1.43.0
```

Comment:
can you add this to the PR description please? :)

Reply:
🚀